### PR TITLE
[docs] Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by ronny-mysten**
## Description 

Describe the changes or additions included in this PR.

## Test Plan 

How did you test the new or updated feature?

---
If your changes are not user-facing and do not break anything, you can skip the following section. Otherwise, please briefly describe what has changed under the Release Notes section.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
+Thanks for considering making a contribution to the Sui network or its documentation. The Sui monorepo houses the code that powers and documents the Sui blockchain. 
 
-See [Contributing to Sui](doc/src/contribute/index.md) for instructions on helping Sui grow.
+## Contribute to Sui
 
-See [ISSUES.md](./ISSUES.md) for filing bug reports.
+Have an idea for a feature that will improve Sui for everyone? Start with a [Sui Improvement Proposal](https://github.com/sui-foundation/sips/tree/main).
 
-See [CONTRIBUTING.md](./crates/sui-framework/CONTRIBUTING.md) for information related to `sui-framework` crate contributions..
+Found a bug or security vulnerability? Create a [GitHub issue](https://github.com/MystenLabs/sui/issues/new/choose).
```
Comment (mlegner) [inline CONTRIBUTING.md:9]: Question: There is currently nothing about contributing code. Is that intentional?

**Comment by mlegner** [comment]
@ronny-mysten I just noticed that the link in the `CODE_OF_CONDUCT.MD` file to the documentation is also broken. Maybe we can fix that as well? 🙂

File: CONTRIBUTING.md (hunk 1 above)
Comment (ronny-mysten) [reply CONTRIBUTING.md:11]: I believe following the SIP process is the recommended approach before contributing code. Your point is valid, though, for smaller code enhancements/fixes. Updated for clarity and addtl info.

File: sdk/typescript/README.md [hunk 2]
```diff
@@ -4,8 +4,8 @@ For more complete docs, visit the [Sui TypeScript SDK docs](https://sdk.mystenla
 
 # Sui TypeScript SDK
 
-This is the Sui TypeScript SDK built on the Sui
-[JSON RPC API](https://github.com/MystenLabs/sui/blob/main/doc/src/build/json-rpc.md). It provides
+This is the Sui TypeScript SDK built on the Sui [JSON RPC
+API](https://github.com/MystenLabs/sui/blob/main/docs/content/references /sui-api.mdx). It provides
```
Comment (mlegner) [inline sdk/typescript/README.md:8]: I believe there's an extra whitespace in the link (also below):
```suggestion
API](https://github.com/MystenLabs/sui/blob/main/docs/content/references/sui-api.mdx). It provides
```

File: sdk/typescript/README.md [hunk 3]
```diff
@@ -16,9 +16,9 @@ upcoming TestNet launch.
 ## Working with DevNet
 
 The SDK will be published to [npm registry](https://www.npmjs.com/package/@mysten/sui.js) with the
-same bi-weekly release cycle as the DevNet validators and
-[RPC Server](https://github.com/MystenLabs/sui/blob/main/doc/src/build/json-rpc.md). To use the SDK
-in your project, you can do:
+same bi-weekly release cycle as the DevNet validators and [RPC
+Server](https://github.com/MystenLabs/sui/blob/main/docs/content/references /sui-api.mdx). To use
```
Comment (mlegner) [inline sdk/typescript/README.md:20]:
```suggestion
Server](https://github.com/MystenLabs/sui/blob/main/docs/content/references/sui-api.mdx). To use
```

File: sdk/typescript/README.md [hunk 4]
```diff
@@ -37,8 +37,8 @@ contain the latest changes from `main`.
 npm install @mysten/sui.js@experimental
 ```
 
-Refer to the [JSON RPC](https://github.com/MystenLabs/sui/blob/main/doc/src/build/json-rpc.md) topic
-for instructions about how to start a local network and local RPC server.
+Refer to the [JSON RPC](https://github.com/MystenLabs/sui/blob/main/docs/content/references
+/sui-api.mdx) topic for instructions about how to start a local network and local RPC server.
```
Comment (mlegner) [inline sdk/typescript/README.md:41]:
```suggestion
Refer to the [JSON RPC](https://github.com/MystenLabs/sui/blob/main/docs/content/references/sui-api.mdx)
topic for instructions about how to start a local network and local RPC server.
```

**Review by mlegner — APPROVED** [review]

**Comment by ronny-mysten** [comment]
> I believe there's an extra whitespace in the link (also below)
I was testing you. You passed.

**Pull request merged**